i18n/.js/src/main/scala/io/udash/i18n/bindings/TranslationBinding.scala

```diff
@@ -2,32 +2,33 @@ package io.udash.i18n.bindings
 
 import com.avsystem.commons._
 import io.udash.bindings.Bindings
-import io.udash.i18n.Translated
+import io.udash.bindings.modifiers._
+import io.udash.i18n._
 import io.udash.logging.CrossLogging
 import org.scalajs.dom._
 import scalatags.JsDom.Modifier
 
 import scala.concurrent.Future
 import scala.util.{Failure, Success}
 
-private[i18n] class TranslationBinding(translation: Future[Translated], placeholder: Option[Element])
-  extends Modifier with Bindings with CrossLogging {
+private[i18n] final class TranslationBinding(
+  translation: Future[Translated],
+  placeholder: Option[Element],
+  rawHtml: Boolean
+) extends Modifier with Bindings with CrossLogging {
 
-  override def applyTo(t: Element): Unit =
-    if (translation.isCompleted && translation.value.get.isSuccess) {
-      t.appendChild(document.createTextNode(translation.value.get.get.string))
-    } else {
-      val holder: Node = placeholder.getOrElse(emptyStringNode())
-      t.appendChild(holder)
+  override def applyTo(t: Element): Unit = translation.value match {
+    case Some(Success(Translated(text))) =>
+      parseTranslation(rawHtml, text).foreach(t.appendChild)
 
+    case _ =>
+      val holder: Seq[Node] = t.appendChild(placeholder.getOrElse(emptyStringNode()))
+      holder.foreach(t.appendChild)
```

Comment:
I think you don't need to append it again - you already do in the line above.

Reply:
Yep, it was a wrong copy and paste